gulpfile.js

```diff
@@ -0,0 +1,142 @@
+var autoprefixer = require("gulp-autoprefixer");
+var changed = require("gulp-changed");
+var connect = require("gulp-connect");
+var eslint = require("gulp-eslint");
+var gulp = require("gulp");
+var gutil = require("gulp-util");
+var less = require("gulp-less");
+var minifyCSS = require("gulp-minify-css");
+var path = require("path");
+var uglify = require("gulp-uglify");
+var webpack = require("webpack");
+
+var dirs = {
+  src: "./src",
+  js: "./src/js",
+  dist: "dist",
+  styles: "./src/css",
+  img: "./src/img",
+  imgDist: "img"
+};
+
+var files = {
+  mainJs: "main",
+  mainJsDist: "main",
+  mainLess: "main",
+  mainCssDist: "main",
+  index: "index.html"
+};
+
+var webpackConfig = {
+  entry: dirs.js + "/" + files.mainJs + ".jsx",
+  output: {
+    path: path.resolve(dirs.dist),
+    filename: files.mainJsDist + ".js"
+  },
+  module: {
+    loaders: [
+      {
+        test: /\.jsx$/,
+        loader: "babel-loader",
+        exclude: /node_modules/
+      }
+    ],
+    postLoaders: [
+      {
+        loader: "transform?envify"
+      }
+    ]
+  },
+  resolve: {
+    extensions: ["", ".jsx", ".js"]
+  }
+};
+
+// Use webpack to compile jsx into js,
+gulp.task("webpack", function (callback) {
+  // Extend options with source mapping
+  if (process.env.GULP_ENV === "development" &&
```

Comment:
We're moving away from broccoli :p 
If it provides value then let's use it, but it's nice when looking at environment configurations and just seeing `NODE_ENV`